test/dotnet.Tests/TelemetryCommandTest.cs

```diff
@@ -19,6 +19,7 @@ public void TrackEvent(string eventName, IDictionary<string, string> properties,
         {
             EventName = eventName;
         }
+		public void Finish();
```

Comment:
Well, actually, it will never compile without a body... so it's not just about code style :trollface: